olp-cpp-sdk-dataservice-read/tests/CatalogRepositoryTest.cpp

```diff
@@ -91,8 +98,7 @@ class CatalogRepositoryTest : public ::testing::Test {
     settings_.network_request_handler = network_;
     settings_.cache = cache_;
 
-    lookup_client_ =
-        std::make_shared<olp::client::ApiLookupClient>(kHrn, settings_);
+    lookup_client_ = std::make_shared<client::ApiLookupClient>(kHrn, settings_);
```

Comment:
There is a distinct using for this one.
```suggestion
    lookup_client_ = std::make_shared<ApiLookupClient>(kHrn, settings_);
```